### PR TITLE
HTML-Tables - Frank Bishop

### PR DESCRIPTION
**Pull request by FrankBishop**
HTML Tables

### CODE REVIEW

File: html-tables/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      <thead>
+        <tr>
+          <th><strong>Student Name</strong></th>
+          <th>Course</th>
+          <th>Grade</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <th>Grace Hopper</th>
```
Comment (codingcodymiller) [inline html-tables/index.html:18]: You should only use `th` for table headers. This is data in the body of the table, so it should be in a `td` element

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-tables/index.html (hunk 1 above)
Comment (FrankBishop) [reply html-tables/index.html]: tables corrected
Sorry about that, I got it fixed.  I was wondering why the rows weren't aligning like they did in the example.

File: html-tables/index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<html>
+  <head>
+    <title>HTML Tables</title>
+  </head>
+  <body>
+    <h1>Student Grade Table</h1>
+    <table>
+      <thead>
+        <tr>
+          <td>Student Name</td>
```
Comment (codingcodymiller) [inline html-tables/index.html:11]: Since these are in a `thead`, they should be `th` elements.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-tables/index.html (hunk 2 above)
Comment (FrankBishop) [reply html-tables/index.html]: tables corrected
It should be correct now, thanks.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**